api/controllers/StatusController.js

```diff
@@ -0,0 +1,22 @@
+const statusAPI = sails.hooks['apistatushook'];
+
+const isDatabaseAvailable = () => {
+  return statusAPI.getStatus();
+};
+
+const getPackageVersion = () => {
+  // 'npm_package_version' is only available when the server is started via an npm script
+  return process.env.npm_package_version || 'unknown';
```

Comment:
Since we have access to internal files, I'm curious if you could just import the package.json and pluck off only the version key so that it's always available. Something like this:
```
const { version } = require('../../../package.json');
```
That path might not be right but the spirit is there.